config/schedule.rb

```diff
@@ -1,5 +1,9 @@
 # Learn more: http://github.com/javan/whenever
-job_type :envcommand, 'cd :path && RAILS_ENV=:environment :bundle_command :task :output'
+
+# These define jobs that checkin with Honeybadger.
+# If changing the schedule of one of these jobs, also update at https://app.honeybadger.io/projects/77112/check_ins
+job_type :rake_rb,
```

Comment:
I think this copypasta may be broken (because it bit us in 1-2 other spots already). The `job_type` is named `rake_rb` here and invoked below as `rake_hb`. Changing this line to `rake_hb` should fix things up.

Reply:
oops, thanks, pulled from a PR in another project, I guess it got fixed in a later PR

Reply:
I did the same thing!